q2cli/core/usage.py

```diff
@@ -199,9 +199,11 @@ def get_metadata_column(self, name, column_name, variable):
         return var
 
     def view_as_metadata(self, name, variable):
-        # super() for validation, otherwise just pass variable through
-        super().view_as_metadata(variable.name, variable)
-        return variable
+        var = super().view_as_metadata(variable.name, variable)
+        # preserve the orginal interface name of the QZA as this will be
```

Comment:
```suggestion
        # preserve the original interface name of the QZA as this will be
```

Reply:
boy I cannot spell for the life of me, I already fixed another typo